elastic/logstash/pipeline/logstash.conf

```diff
@@ -0,0 +1,140 @@
+input {
+  file {
+    path => "/app/logstash/monitor/*.log"
+    start_position => "beginning"
+  }
+}
+
+
+filter {
+
+  # create hash from message to handle duplicates
+  fingerprint {
+    source => "message"
+    target => "[@metadata][fingerprint]"
+    method => "MURMUR3"
+  }
+
+
+  if [fileset][module] == "nginx" {
+    if [fileset][name] == "access" {
+      grok {
+        match => { "message" => ["%{IPORHOST:[nginx][access][remote_ip]} - %{DATA:[nginx][access][user_name]} \[%{HTTPDATE:[nginx][access][time]}\] \"%{WORD:[nginx][access][method]} %{DATA:[nginx][access][url]} HTTP/%{NUMBER:[nginx][access][http_version]}\" %{NUMBER:[nginx][access][response_code]} %{NUMBER:[nginx][access][body_sent][bytes]} \"%{DATA:[nginx][access][referrer]}\" \"Bearer %{DATA:[@metadata][jwt]}\" \"%{DATA:[nginx][access][agent]}\" \"%{IPORHOST:[nginx][access][client_ip]}\" \"%{GREEDYDATA:[nginx][access][payload]}\""] }
```

Comment:
We have a ticket right now (MDS-1619) that will be adding groups. Not sure if this will bring that JWT item in.